devel/mxml/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        michaelrsweet mxml 3.1 v
+
+categories          devel textproc
+platforms           darwin
+license             Apache-2.0
+maintainers         nomaintainer
+
+description         Mini-XML is a tiny XML library that you can use to read and write \
+                    XML and XML-like data files in your application without requiring \
+                    large non-standard libraries.
+long_description    ${description} Mini-XML only requires an ANSI C compatible compiler \
```

Comment:
You should use the expand operator here: `{*}${description}`

Though if you shorten the short description to a sentence fragment you'll want to change the long description to account for that.